src/utils/helpTextForForms.tsx

```diff
@@ -168,4 +168,10 @@ export const lableTypeFormHelpText: HelpText = {
       <p><i>If you are not an administrator, this field is always pre-filled with the current organisation.</i></p>
     </>
   ),
+}
+
+export const personalApiKeysFormHelpText: HelpText = {
+  default: "Personal API keys can be used to authenticate external applications in Lizard",
+  name: "Name of the personal api key",
+  scope: "Defines what the personal api key can be used for",
```

Comment:
This help text for scope field cannot be shown right now because the field name is different.

Reply:
fixed here:
https://github.com/nens/lizard-management-client/pull/596